packages/hub/main.ts

```diff
@@ -79,58 +85,74 @@ export function wireItUp(registryCallback?: RegistryCallback): Container {
   return new Container(registry);
 }
 
-export async function makeServer(registryCallback?: RegistryCallback, containerCallback?: ContainerCallback) {
-  let container = wireItUp(registryCallback);
-  containerCallback?.(container);
+const LOGGER = logger('hub/server');
+
+// Empty for now
+const DEFAULT_CONFIG: ServerConfig = {};
+export class HubServer {
+  logger = LOGGER;
+  static logger = LOGGER;
+
+  static async create(config?: ServerConfig): Promise<HubServer> {
+    let container = wireItUp(config?.registryCallback);
+    config = Object.assign({}, DEFAULT_CONFIG, config);
+
+    initSentry();
+
+    let app = new Koa<Koa.DefaultState, Koa.Context>()
+      .use(CardstackError.withJsonErrorHandling)
+      .use(environment)
+      .use(cors({ origin: '*', allowHeaders: 'Authorization, Content-Type, If-Match, X-Requested-With' }))
+      .use(httpLogging);
+
+    app.use(((await container.lookup('authentication-middleware')) as AuthenticationMiddleware).middleware());
+    app.use(((await container.lookup('development-proxy-middleware')) as DevelopmentProxyMiddleware).middleware());
+    app.use(((await container.lookup('api-router')) as ApiRouter).routes());
+    app.use(((await container.lookup('callbacks-router')) as CallbacksRouter).routes());
+
+    function onError(err: Error, ctx: Koa.Context) {
```

Comment:
@mattmcmanus it's a little tiny thing, but since you are already in there, could you add this? It's a huge quality of life improvement when unexpected errors occur:
```js 
logger.error(`Unhandled error:`, err);
```

Reply:
Added